worker/include/Utils.hpp

```diff
@@ -266,10 +266,10 @@ namespace Utils
 
 			for (size_t i{ 0 }; i < len; ++i)
 			{
-				buffer[i] = chars[GetRandomUInt(0, sizeof(chars) - 1)];
+				buffer[i] = Chars[GetRandomUInt(0, sizeof(Chars) - 1)];
 			}
 
-			return std::string(buffer, len);
+			return { buffer, len };
```

Comment:
What is this?

Reply:
The method is returning a `string`, this is creating the `string` with the given initialization list.

Reply:
Got it.